### DISCUSSION TITLE
使用的5.16.4版本，Table 虚拟列表同样的scroll.x,我的横向滚动条比官方示例的小很多

### DISCUSSION THREAD

**Discussion by xiaobing0723** [ant-design/ant-design#49096] [Q&A] [answered]
![微信图片_20240528164613](https://github.com/ant-design/ant-design/assets/134380792/7fb9ab96-3cb2-469f-bcd6-25d516416994)
使用虚拟滚动时横向滚动条宽33px，正常列表横向滚动条366px

**Comment by linxianxi** [comment, marked as answer]
删除 lock，重新安装

**Reply by yoyo837** [reply]
node_modules也得删

**Reply by xiaobing0723** [reply]
删除lock和node_modules确实可以，但是本地运行的时候控制台报错
![微信图片_20240528173822](https://github.com/ant-design/ant-design/assets/134380792/44005011-cec9-4170-920c-327612fe8b3a)
![微信图片_20240529110743](https://github.com/ant-design/ant-design/assets/134380792/a02599d4-6b92-4b5a-a058-41ead4e0816c)

**Reply by xiaobing0723** [reply]
我把antd相关的lock删除，重新安装，可以了，多谢多谢
